Bottom: a2686301246303828c867b2d2423a1b889794717
Top:    881f45a8fcf66a6edc0a241145c101d948b62869
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-06 00:34:19 +0100

Refresh of add-a-hid-api-to-notify-the-gu

---

diff --git a/src/action.c b/src/action.c
index c375805..f484d31 100644
--- a/src/action.c
+++ b/src/action.c
@@ -5685,7 +5685,7 @@ ActionSaveTo (int argc, char **argv, Coord x, Coord y)
       if (SavePCB (name) == 0)
         {
           if (gui->notify_pcb_filename_change != NULL)
-            gui->notify_pcb_filename_changed (PCB->Filename, name);
+            gui->notify_pcb_filename_change (PCB->Filename, name);
           free (PCB->Filename);
           PCB->Filename = strdup (name);
           SetChangedFlag (false);
diff --git a/src/command.c b/src/command.c
index 4ed7ef5..584cc56 100644
--- a/src/command.c
+++ b/src/command.c
@@ -352,7 +352,7 @@ CommandSaveLayout (int argc, char **argv, Coord x, Coord y)
       if (SavePCB (argv[0]) == 0)
         {
            if (gui->notify_pcb_filename_change != NULL)
-            gui->notify_pcb_filename_changed (PCB->Filename, argv[0]);
+            gui->notify_pcb_filename_change (PCB->Filename, argv[0]);
           free (PCB->Filename);
           PCB->Filename = strdup (argv[0]);
           SetChangedFlag (false);
